source/nodejs/adaptivecards/src/strings.ts

```diff
@@ -70,7 +70,9 @@ export class Strings {
         elementAlreadyParented: () => "The element already belongs to another container.",
         actionAlreadyParented: () => "The action already belongs to another element.",
         elementTypeNotStandalone: (typeName: string) =>
-            `Elements of type ${typeName} cannot be used as standalone elements.`
+            `Elements of type ${typeName} cannot be used as standalone elements.`,
+        invalidInputLabelWidth: () =>
+            `Invalid input label width. Defaulting to label width from host config"`,
```

Comment:
Period at end for consistency with other strings.

Reply:
done